docs/README.md

```diff
@@ -28,6 +28,7 @@ Here you can find the complete list of decorators, however we encourage you to r
   - [`@injectProps`](functions/injectProps.md)
   - [`@injectState`](functions/injectState.md)
   - [`@debounce(wait: number = 300, immediate: boolean = false)`](functions/debounce.md)
+	- [`@lock`](functions/lock.md)
```

Comment:
Here the new decorator is defined as `@lock` but it is actually `@lock()`.
If we don't need parameters we can just change its implementation in `@lock`, simplifying the API. What do you think?

Reply:
That sounds good! I changed it.